SQL Query/get_open_connections.sql

```diff
@@ -0,0 +1,3 @@
+SELECT DB_NAME(dbid) as DBName, COUNT(dbid) as NumberOfConnections, loginam as LoginName, sum(cpu) CPU, hostname 
+FROM sys.sysprocessesWHEREdbid > 0
+--and login_time > '2022-08-23' and login_time < '2022-08-24'GROUP BYdbid, loginame,hostname;
```

Comment:
I have reviewed your query and would suggest the following changes:

- [ ] please convert all SQL keywords into uppercase
- [ ] break the query into multiple lines logic-wise instead of a big line
- [ ] remove comments
- [ ] generic variable names
- [ ] add description like other queries in the repo

Reply:
TODO in future PR